content/introduction-and-setup/walkthrough/mac-install/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Mac Users: Visual Studio for Mac"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 3
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Before you start the installation guide, read through each step below. You will find the recommended features and functionality to install listed in each step.
+
+{{% notice orange "Warning" "rocket" %}}
+Allow at least **an hour** for the installation process.
+{{% /notice %}}
+
+## Installation Steps
+
+### Prerequisites
+
+Before you start installing anything, you should first make sure that you can install this on your computer.
+
+### Install Visual Studio
+
+Walk through the installation process by following Microsoft’s [Install Visual Studio for Mac](https://learn.microsoft.com/en-us/visualstudio/mac/installation?view=vsmac-2022) guide.
+
+As you install Visual Studio, keep these things in mind:
+
+1. You will have to assist the installer as needed. This may take a while, but stick around. You may be asked to enter your password a few times.
+1. To personalize or not? This is up to you.
+
+   1. Microsoft Account: Creating a Microsoft account is optional.
```

Comment:
When I opened VS recently I was prompted to make a free account and sign in to Microsoft to keep using Visual Studio for Mac for free.  The wording was that my free trial was coming to an end.  My take away was that I needed the account to keep using VS, still free but must have account.

Reply:
Addressed this with some updated text.

Reply:
Thanks Sally!